Update good module practices (#1127)
PrestaShop/docs: modules/creation/good-practices.md

- Don't go through directories using code and variables like: `dirname(__FILE__).'/../../config/config.inc.php'`
- Don't edit the SQL structure of PrestaShop tables.
- Don't obfuscate your code, making it not human readable.

> - Don't use global variables to avoid any conflict and because it's a bad practice.

Reviewer: ```suggestion
- Don't use global variables to avoid any conflict. Also global variables are usually considered a bad practice.
```
